src/main/java/model/data/value/TemporalValue.java

```diff
@@ -13,6 +13,13 @@
 
 	private String format;
 
+	/**
+	 * Return a null instance.
+	 */
+	TemporalValue() {
+		this(null);
```

Comment:
In alle datavalues zie ik dat je op deze manier voor lege input een null returnt. Om alle andere functionaliteiten rond null dan te laten functioneren moet je isNull op true zetten. 
setNull(true); moet overal dus bij